pkgs/tools/package-management/nix-top/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, ruby
+, makeWrapper
+, getent               # /etc/passwd
+, ncurses              # tput
+, binutils-unwrapped   # strings
+, coreutils
+, findutils
+}:
+
+# No gems used, so mkDerivation is fine.
+let
+  additionalPath = lib.makeBinPath [ getent ncurses binutils-unwrapped coreutils findutils ];
+in
+stdenv.mkDerivation rec {
+  pname = "nix-top";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "jerith666";
+    repo = "nix-top";
+    rev = "v${version}";
+    sha256 = "sha256-w/TKzbZmMt4CX2KnLwPvR1ydp5NNlp9nNx78jJvhp54=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    ruby
+  ];
+
+  installPhase = ''
```

Comment:
`installPhase` hooks are missing

Reply:
I don't understand what you're getting at here, can you give more details?

Reply:
You are overwriting the install phase but not re-adding the hooks which is going to lead to preInstallPhase and postInstallPhase (and possible something else) to fail.
runHook is necessary.

Reply:
https://discourse.nixos.org/t/what-does-runhook-do/13861

Reply:
Okay, I think I see ... look good now?